### DISCUSSION TITLE
federated versus centralized storage

### DISCUSSION THREAD

**Discussion by tngTUDOR** [Depart-de-Sentier/sentier.dev#10] [General]
In both cases, (federated and centralized)  storage, we can achieve some **desirable** properties for storage such as:

+ parallel access
+ redundancy
+ reliability
+ scalability

Also, modern orchestration (of containers) tools enable the use of many different nodes, that appear as a single entry point, with the advantage of having on-demand creation of new nodes to provide a service

[ILCDN](https://eplca.jrc.ec.europa.eu/LCDN/contactListEF.html) (from PEF) as well as [GLAD](https://www.globallcadataaccess.org/search) are examples of "federated" nodes providing data (not models). 

Does it make any difference to store "models" and to store data? Are there any specific properties from these two types of assets that would make having a different (federated or centralized) storage type ?

Choosing a paradigm (federated or centralized) right now, can slow-down future developments. Is it safer to go with centralized for the moment and move to federated later on, or the other way around?

**Comment by cmutel** [comment]
**Current Decision**: To get reasonable performance we can't have data calls taking 100 msec, so we go with centralized storage, but make it easy to have mirrors of the database.
